src/components/formProducts/FormProducts.js

```diff
@@ -52,13 +47,8 @@ const FormProducts = ({ items }) => {
         </div>
       </form>
       {itemsFiltered?.length > 0 ? (
-        itemsFiltered.map((item, index) => (
-          <ProductForList
-            key={`${index}${item.name}`}
-            item={item}
-            handleDeleteAttempt={handleDeleteAttempt}
-            token={token.current}
-          />
+        itemsFiltered.map((item) => (
+          <ProductForList item={item} token={token.current} />
```

Comment:
```suggestion
          <ProductForList
            key={`${index}-${item.name}`}
            item={item}
            token={token.current}
          />
```

Podrían "revertir" el cambió que tenían previamente 😄 , donde tenía el `key`.

Reply:
Listo por aqui https://github.com/the-collab-lab/tcl-39-smart-shopping-list/commit/fc298dc0b2ff55be42c6d4b3657067b98ef3fd8d